model/common_datatypes.smithy

```diff
@@ -0,0 +1,14 @@
+namespace OpenSearch
```

Comment:
Need to add copyright comment section for each file

Reply:
SHould this copyright be similar to the one included in the gradle file? Or will it have a different syntax?

Reply:
similar to the one in gradle file